src/resmom/start_exec.c

```diff
@@ -3865,7 +3865,11 @@ finish_exec(job *pjob)
 				}
 			}
 
-			mom_writer(qsub_sock, ptc);
+			int res = mom_writer(qsub_sock, ptc);
+			if (res == -1)
+				log_eventf(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_ERR, pjob->ji_qs.ji_jobid, "CS_write failed with errno %d", errno);
```

Comment:
shouldn't this be DEBUG3?

Reply:
> shouldn't this be DEBUG3?

Inside mom_writer and mom_reader function, if read is successful and write fails then it is an error and that is why I am using PBSEVENT_ERROR here instead of PBSEVENT_DEBUG3.